### ISSUE TITLE
"Same line" statements belonging to "else" bodies are not reported

### ISSUE THREAD

**Issue by cgewecke** [JoinColony/solcover#38]
Istanbul reports a taken else branch as follows
![ifelse](https://cloud.githubusercontent.com/assets/7332026/21837925/fb197d98-d783-11e6-80bb-aa10d1773ca2.png)

Solcover reports:
![elsesol](https://cloud.githubusercontent.com/assets/7332026/21837972/46d23d88-d784-11e6-9ba4-205783cc6c96.png)

The instrumentation looks like this:
```
Coverage('/Users/user/Sites/sol-coveralls/solcover/test.sol',5);
         StatementCoverage('/Users/user/Sites/sol-coveralls/solcover/test.sol',1);
if (x == 1) {BranchCoverage('/Users/user/Sites/sol-coveralls/solcover/test.sol',1,0);  StatementCoverage('/Users/user/Sites/sol-coveralls/solcover/test.sol',2);
x+= 1;}
        else {BranchCoverage('/Users/user/Sites/sol-coveralls/solcover/test.sol',1,1);  StatementCoverage('/Users/user/Sites/sol-coveralls/solcover/test.sol',3);
x += 5;}
    }
```
Same as #37.

**Comment by cgewecke** [comment]
@area I feel like this same-line `else` construction is relatively common and Istanbul's report is intuitively more informative. But as you note in #37  - the obvious fix would produce `11` hits. Also bound up with the `else if` analysis you made in the highlighting issue. . . . not sure what should be done here.

**Comment by area** [comment]
I agree that our output here definitely could be better. I'll have a fiddle!
